frontend/src/libs/components/header/styles.module.scss

```diff
@@ -1,5 +1,5 @@
 .header {
-  position: relative;
+  position: absolute;
```

Comment:
can we use fixed here?

Reply:
Yes, we used fixed early. But it was changed for another task (I fixed it too)